modules/wmco-prerequisites.adoc

```diff
@@ -85,16 +85,22 @@ The following table lists the supported link:https://docs.microsoft.com/en-us/wi
 |Supported Windows Server version
 
 |Amazon Web Services (AWS)
-|Windows Server Long-Term Servicing Channel (LTSC): Windows Server 2019
+|Windows Server 2019
```

Comment:
Please specify `Windows Server 2019, version 1809` — there’s multiple Windows Server 2019 version and this is the only one we support. Applies for all platforms 2019 is supported

Reply:
@saifshaikh48 Windows Server 2019, version 1809 for all references?